### PR TITLE
Delete list dialogue box and create list window

### PR DESCRIPTION
**Pull request by vader-sama**
Both are only templates, they don't provide any functionality.

### CODE REVIEW

File: src/components/Dashboard/Selected/DeleteList.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+import Window from "../../../util/Window";
+
+//Description of the dialouge
```
Comment (jenova-0) [inline src/components/Dashboard/Selected/DeleteList.js:5]: Fix the typo in the comment

**Review by jenova-0 — CHANGES_REQUESTED** [review]
Apply requested changes. 👻

File: src/components/Dashboard/Selected/DeleteList.js (hunk 1 above)
Comment (vader-sama) [reply src/components/Dashboard/Selected/DeleteList.js]: Fixed in 52e385a.

**Review by jenova-0 — APPROVED** [review]
LGTM! 😅👽

**Pull request merged**